product/r7insight.js

```diff
@@ -392,29 +396,30 @@
         createLogStream: _createLogStream,
         to: _getLogger,
         destroy: _destroyLogStream,
-        log: function() {
+        log: function () {
             for (var k in loggers)
                 loggers[k].log.apply(this, arguments);
         },
-        warn: function() {
+        warn: function () {
             for (var k in loggers)
                 loggers[k].warn.apply(this, arguments);
         },
-        error: function() {
+        error: function () {
             for (var k in loggers)
                 loggers[k].error.apply(this, arguments);
         },
-        info: function() {
+        info: function () {
             for (var k in loggers)
                 loggers[k].info.apply(this, arguments);
         }
     };
 
     function validate_region(region) {
-        var allowed_regions = ['eu', 'us', 'us2', 'us3', 'ca', 'au', 'ap'];
+        var allowed_regions = ['eu', 'us', 'us2', 'us3', 'ca', 'au', 'ap','custom'];
         if (region) {
             if (allowed_regions.indexOf(region) > -1) {
-                return region;
+                if(region == 'custom') return "";
```

Comment:
triple equals check instead of double and can be a ternary. And mix of double and single quotes here.

return `region === 'custom' ? '' : region + '.'`

Reply:
Honestly I was just trying to touch as little of its behaviour as possible... like R7 has done since copy pasting the repo over from LE.